### ISSUE TITLE
Support cert-manager 1.7 and above

### ISSUE THREAD

**Issue by pradithya** [kserve/kserve#2029]
/kind feature

**Describe the solution you'd like**
cert-manager 1.7 removed several deprecated APIs listed [here](https://cert-manager.io/docs/installation/upgrading/remove-deprecated-apis/)

Installing kserve and while using cert-manager 1.7 causes following error
```
unable to recognize "kserve.yaml": no matches for kind "Certificate" in version "cert-manager.io/v1alpha2"
unable to recognize "kserve.yaml": no matches for kind "Issuer" in version "cert-manager.io/v1alpha2"
```

**Anything else you would like to add:**
[Miscellaneous information that will assist in solving the issue.]

**Comment by yuzisun** [comment]
@pradithya This is fixed in the upcoming v0.8.0 release, see https://github.com/kserve/kserve/pull/1904

**Comment by pradithya** [comment]
Thanks @yuzisun , any expected date for 0.8 release?
